elementary/monitor/data_monitoring/schema.py

```diff
@@ -101,6 +104,83 @@ def validate_report_selector(self):
                 "Selector is invalid for report: ", self.selector
             )
 
+    @staticmethod
+    def from_cli_params(cli_filters: Tuple[str]) -> "FiltersSchema":
+        if not cli_filters:
+            return FiltersSchema()
+
+        tags = []
+        owners = []
+        models = []
+        statuses = []
+        resource_types = []
+
+        for filter in cli_filters:
+            tags_match = FiltersSchema._match_filter_regex(
+                filter_string=filter, regex=re.compile(r"tags:(.*)")
+            )
+            if tags_match:
+                tags.append(FilterSchema(values=tags_match))
+                continue
+
+            owners_match = FiltersSchema._match_filter_regex(
+                filter_string=filter, regex=re.compile(r"owners:(.*)")
```

Comment:
instead of using re.compile, re.search and groups, you can simply use `re.findall("models:(\w+)", filter)`. this returns a list, and if there's no match it's an empty list. will save you a lot of code